ironfish/src/mining/poolDatabase/database.test.ts

```diff
@@ -21,7 +21,7 @@ describe('poolDatabase', () => {
     const fileSystem = new NodeFileProvider()
     await fileSystem.init()
     // TODO(mat): It would be convenient if we didn't need a filesystem for Config for tests
-    const config = new Config(fileSystem, dataDir)
+    const config = Config.Init(fileSystem, dataDir)
```

Comment:
should this be `Config.NewConfig`? FWIW I think I prefer the syntax of something like `Config.new` or `Config.init`

Reply:
Just fixed this, but it actually revealed that the solution doesn't work without a cast.